Source/MediaInfo/MediaInfo_Inform.cpp

```diff
@@ -854,6 +855,24 @@ Ztring MediaInfo_Internal::Inform (stream_t StreamKind, size_t StreamPos, bool I
                     }
                 }
 
+                //Handling ISO-6709 location data from Android devices (latitide and logitude only) for Text and HTML display
+                bool TextOrHTML = false;
+                #if defined(MEDIAINFO_TEXT_YES) && (defined(MEDIAINFO_HTML_YES) || defined(MEDIAINFO_XML_YES) || defined(MEDIAINFO_CSV_YES))
+                if (Text) TextOrHTML = true;
+                #else
+                TextOrHTML = true;
+                #endif // defined(MEDIAINFO_TEXT_YES) && (defined(MEDIAINFO_HTML_YES) || defined(MEDIAINFO_XML_YES) || defined(MEDIAINFO_CSV_YES))
+                #if defined(MEDIAINFO_HTML_YES)
+                if (HTML) TextOrHTML = true;
+                #endif // defined(MEDIAINFO_HTML_YES)
+                if ((TextOrHTML) && (Get(StreamKind, StreamPos, Champ_Pos, Info_Name) == __T("Location"))) {
+                    std::string ISO6709{ Valeur.To_UTF8() };
+                    std::smatch match;
+                    std::regex pattern(R"((([-+]?\d{2})(?:\.\d+)?)(([-+]\d{3})(?:\.\d+)?)/)");
+                    if (std::regex_search(ISO6709, match, pattern))
+                        Valeur.From_UTF8(match[1].str() + ", " + match[3].str());

```

Comment:
", " --> " ", it is also supported by Google Maps.
With depth and CRS, I would output e.g. "+27.5916 +086.5640 +8850 WGS_84".